piazza_api/network.py

```diff
@@ -173,6 +173,26 @@ def create_instructor_answer(self, post, content, revision, anonymous=False):
         }
         return self._rpc.content_instructor_answer(params)
 
+    def mark_as_duplicate(self, duplicated_cid, master_cid, msg=''):
+        """Mark the post at `duplicated_cid` as a duplicate of `master_cid`
```

Comment:
Actually everywhere else in the file, all the doc strings uses single backtick to reference variables. I think the style should be consistent through out the file